commit 28f3050b13ab69e01e435962c298b345ede591ed
Author: Adam Ford <aford173@gmail.com>
Date:   Sun Oct 28 15:34:21 2018 -0500

    ARM: dts: am3517-som: Fix WL127x Wifi interrupt
    
    [ Upstream commit 419b194cdedc76d0d3cd5b0900db0fa8177c4a52 ]
    
    At the same time the AM3517 EVM was gaining WiFi support,
    separate patches were introduced to move the interrupt
    from HIGH to RISING.  Because they overlapped, this was not
    done to the AM3517-EVM.  This patch fixes Kernel 4.19+
    
    Fixes: 6bf5e3410f19 ("ARM: dts: am3517-som: Add WL127x Wifi")
    
    Signed-off-by: Adam Ford <aford173@gmail.com>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/am3517-som.dtsi b/arch/arm/boot/dts/am3517-som.dtsi
index dae6e458e59f..b1c988eed87c 100644
--- a/arch/arm/boot/dts/am3517-som.dtsi
+++ b/arch/arm/boot/dts/am3517-som.dtsi
@@ -163,7 +163,7 @@
 		compatible = "ti,wl1271";
 		reg = <2>;
 		interrupt-parent = <&gpio6>;
-		interrupts = <10 IRQ_TYPE_LEVEL_HIGH>; /* gpio_170 */
+		interrupts = <10 IRQ_TYPE_EDGE_RISING>; /* gpio_170 */
 		ref-clock-frequency = <26000000>;
 		tcxo-clock-frequency = <26000000>;
 	};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1029_28f3050b13ab69e01e435962c298b345ede591ed.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1029_28f3050b13ab69e01e435962c298b345ede591ed.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
